Realm/RLMAPIKeyAuth.h

```diff
@@ -40,7 +40,7 @@ typedef void(^RLMUserAPIKeysBlock)(NSArray<RLMUserAPIKey *> *  _Nullable, NSErro
   @param name The name of the API key to be created.
   @param completion A callback to be invoked once the call is complete.
 */
-- (void)createApiKeyWithName:(NSString *)name
+- (void)createAPIKeyWithName:(NSString *)name
                   completion:(RLMOptionalUserAPIKeyBlock)completion NS_SWIFT_NAME(createApiKey(named:completion:));
```

Comment:
Should we change Swift name too?

Reply:
true

Reply:
Im not sure these are extended in Swift until Pavel's Combine PR.

Reply:
Yes, this is available in both callback and future forms